### PR TITLE
Adds Hiring and Pitching blog

### PR DESCRIPTION
**Pull request by realslimshanky**
This PR changes the following to origin repository

- New blog 'How PyDelhi conducts Hiring and Pitching sessions' added
- `hiringpitching.jpg` added to be used in above blog

Please review. @theskumar @CuriousLearner

### CODE REVIEW

File: content/2018-05-02-how-pydelhi-conducts-hiring-pitching-sessions.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Email: abc@shanky.xyz
+Summary: Hiring and Pitching sessions have been a part of PyDelhi events since the beginning. Here's how it is organized.
+
+![Hiring Pitching Sessions]({filename}/images/hiringpitching.jpg)
+
+Hiring and Pitching sessions have been a part of PyDelhi events since the beginning. The session is scheduled close to the end of every event just before the closing notes. This blog is to help out volunteers and participants by putting organized structure to the entire process. Points raised below are the work of volunteers, open to discussion and will be shaped better with time.
+
+## Step 1 - Broadcasting on social media
+
+The initial step, before even the event start, volunteers should be sharing the link to etherpad which will be used to jot down details of folks speaking during the session. This will help folks to be prepared beforehand and can be called by the volunteers during the session. Also, this will be beneficial for the folks who are not able to attend the event. Official PyDelhi channels (Twitter/FB) should be preferred to broadcast the link. Etherpad link which can be used is https://public.etherpad-mozilla.org/p/Oms9Iw8MK4
```
Comment (shagunsodhani) [inline content/2018-05-02-how-pydelhi-conducts-hiring-pitching-sessions.md]: Reworded the first sentence `The initial step, before even the event start, volunteers should be sharing the link to etherpad which will be used to jot down details of folks speaking during the session.`  to `Before the event starts, volunteers should share a link on etherpad to note down the details of the speakers.`

Comment (shagunsodhani) [inline content/2018-05-02-how-pydelhi-conducts-hiring-pitching-sessions.md]: Maybe consider replacing `folks` by `speakers`, `members`, `audience`, `attendees` etc as appropriate. `folks` sounds too informal.

File: content/2018-05-02-how-pydelhi-conducts-hiring-pitching-sessions.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Step 2 - Conducting Session
+
+Close to the end of the event or by the schedule for Hiring and Pitching session, volunteers should open the etherpad link, preferably with the projector connected and call out the folks who submitted details one by one. If the person is not present, their details should be called out by volunteer themselves. After when the pre-submitted details are over, one of the volunteers should call folks from the audience to see if some of the would-be interested in sharing their details.
```
Comment (shagunsodhani) [inline content/2018-05-02-how-pydelhi-conducts-hiring-pitching-sessions.md]: Maybe consider replacing `call out the folks who submitted details one by one.` by `invite the speakers to make their pitch one by one.`

Comment (shagunsodhani) [inline content/2018-05-02-how-pydelhi-conducts-hiring-pitching-sessions.md]: Maybe consider replacing `to see if some of the would-be interested in sharing their details.` by `to see if some of them would want to make a pitch.`

**Review by shagunsodhani — COMMENTED** [review]
Minor suggestions. Mostly well written 👍

File: content/2018-05-02-how-pydelhi-conducts-hiring-pitching-sessions.md (hunk 2 above)
Comment (realslimshanky) [reply content/2018-05-02-how-pydelhi-conducts-hiring-pitching-sessions.md]: Speakers might be confused by the meetup speakers for the day. `attendees` would sound better here.

File: content/2018-05-02-how-pydelhi-conducts-hiring-pitching-sessions.md (hunk 1 above)
Comment (realslimshanky) [reply content/2018-05-02-how-pydelhi-conducts-hiring-pitching-sessions.md]: ACK.

Comment (realslimshanky) [reply content/2018-05-02-how-pydelhi-conducts-hiring-pitching-sessions.md]: ACK.

File: content/2018-05-02-how-pydelhi-conducts-hiring-pitching-sessions.md (hunk 2 above)
Comment (realslimshanky) [reply content/2018-05-02-how-pydelhi-conducts-hiring-pitching-sessions.md]: ACK.

**Comment by realslimshanky** [comment]
Thank you for the review @shagunsodhani. I've added proposed changes accordingly.

**Review by rajataaron — APPROVED** [review]
Looks good to me. And this blog is necessary for future too Thanks @realslimshanky .

**Comment by rajataaron** [comment]
@CuriousLearner can you merge these changes.

**Comment by CuriousLearner** [comment]
@rajataaron Done :)

**Pull request merged**